services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -141,9 +150,7 @@ async def start_project_interactive_services(
         project["uuid"],
         user_id,
     )
-    running_services = await director_api.get_running_interactive_services(
-        request.app, user_id, project["uuid"]
-    )
+    running_services = await get_services(request.app, user_id, project["uuid"])
```

Comment:
since we are in a big file. I think it would be nice if you have these kind of calls as ```director_v2.get_services```